main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_data in players_data:
+        user = players_data[player_data]
```

Comment:
The variable `user` is assigned the value of `players_data[player_data]`, which is correct. However, the usage of `player_data` as a key might be confusing. Consider renaming `player_data` to something like `player_nickname` to clarify that it represents the player's nickname.